sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala

```diff
@@ -1599,9 +1599,18 @@ private[sql] object QueryExecutionErrors extends QueryErrorsBase {
       s"$commitProtocol does not support adding files with an absolute path")
   }
 
-  def microBatchUnsupportedByDataSourceError(srcName: String): Throwable = {
+  def microBatchUnsupportedByDataSourceError(
+      srcName: String,
+      disabledSources: String,
+      table: Table): Throwable = {
     new UnsupportedOperationException(
-      s"Data source $srcName does not support microbatch processing.")
+      s"""
+         |Data source $srcName does not support microbatch processing.
+         |
+         |It could be the data source is disabled at spark.sql.streaming.disabledV2MicroBatchReaders
```

Comment:
Should better use `SQLConf.get.DISABLED_V2_STREAMING_MICROBATCH_READERS.key`

Reply:
Done